readthedocs_theme/templates/includes/topnav.html

```diff
@@ -118,30 +139,147 @@
             {{ menu_main() }}
 
             <div class="right menu">
-              <div class="ui wide dropdown item" data-module="dropdown" data-action="select" data-display-type="block">
-                <span class="text">Help</span>
-                <i class="fad fa-caret-down icon"></i>
-                <div class="menu">
-                  {{ menu_help() }}
+
+              <div class="item">
+                {# TODO: Add support page to marketing website #}
+                <div class="ui wide dropdown item" data-module="dropdown" data-action="select" data-display-type="block">
+                  <a aria-label="Help" class="ui circular basic icon button">
+                    <i class="fa-solid fa-message-question icon"></i>
+                  </a>
+                  <div class="menu">
+                    {{ menu_help() }}
+                  </div>
```

Comment:
This is an `.item` inside a `.item` class, and is also wrapping the button in a `.ui.dropdown.item`. Both of these are causing the menu item to look like a button in a button.

![image](https://github.com/readthedocs/website/assets/1140183/db8ce661-4c85-4c3a-b73e-5828be8292f2)


Instead, the button should be a `.ui.dropdown.button`. There is an example of a icon dropdown menu in the docs:

https://fomantic-ui.com/modules/dropdown.html#pointing

I didn't test this, but something like:

```suggestion
                <div aria-label="Help" class="ui circular basic icon dropdown button" data-module="dropdown" data-action="select" data-display-type="block">
                  <i class="fa-solid fa-message-question icon"></i>
                  <div class="menu">
                    {{ menu_help() }}
                  </div>
                </div>
```